src/FlapperUniV2SwapOnly.sol

```diff
@@ -0,0 +1,183 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
```

Comment:
When finalizing we need to add this contract to the README.

Reply:
Done.